genesis/engine/scene.py

```diff
@@ -181,6 +182,7 @@ def __init__(
         self._t = 0
         self._is_built = False
         self._show_FPS = show_FPS
+        self._FPS_tracker_alpha = FPS_tracker_alpha
```

Comment:
it's not used?

Reply:
Here:
![Screenshot 2025-06-05 at 6 43 23 PM](https://github.com/user-attachments/assets/4c96005a-95e4-4819-84a7-a817e1630e56)

Reply:
yeah, I mean there is no moving average yet

Reply:
oh, I see, this is already implemented

Reply:
Yes. This PR just makes it configurable.